src/components/entity-panel/sections/entity/EntityCard.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <v-badge
+    :content="noOfEntitiesOnCanvas"
+    :value="noOfEntitiesOnCanvas > 0"
+    color="primaryShade"
+    overlap
+    offset-x="20"
+    offset-y="20"
+    class="custom-entity-card-badge"
+  >
+    <v-card
+      :class="['custom-entity-card my-1', active ? 'custom-entity-card-active' : '']"
+      :data-tier="entity.tier"
+      :data-type="entity.type"
+      :data-image="entity.image"
+      :title="entity.title"
+      elevation="0"
+      @click.stop="onClickHandler"
+    >
+      <v-container class="pa-1">
+        <v-row class="text-center mx-0">
+          <v-col cols="12" class="pa-0 custom-entity-card-column">
+            <img :style="{backgroundColor: entity.color}" :src="entity.image" class="custom-entity-card-image">
+          </v-col>
+          <v-col cols="12" class="pa-0">
+            <div class="caption custom-entity-card-text">{{entity.name}}</div>
+            <v-chip
+              v-if="entity.tier"
+              x-small
+              pill
+              class="ma-0"
+            >T-{{entity.tier}}</v-chip>
```

Comment:
Should we rather use Roman numbers here as games are using them? Also many other games outside of Wargaming are not using tiers, how is that planned?

Reply:
There is an if clause that checks to see if there is a tier property on the entity.
I have created a function to return it as roman numerals instead now

Reply:
![image](https://user-images.githubusercontent.com/24481971/86509192-06a3aa00-bdde-11ea-9cb8-b7352b78a28a.png)

Reply:
Great, basically in this also the `T-X` could be just replaced with `X`, makes also stuff cleaner.

Reply:
Done already